### ISSUE TITLE
[brace-style] 1tbs not enforced with interfaces

### ISSUE THREAD

**Issue by delventhalz** [typescript-eslint/typescript-eslint#804]
**What code were you trying to parse?**

With the following rule in `.eslintrc.json`:
```json
{
  "rules": {
    "brace-style": ["error", "1tbs"]
  }
}
```

This code is _correctly_ flagged as violating 1tbs:
```ts
class Foo
{
  bar:
  {
    baz: string
  } }
```

However, when recreating the same violation with an interface, there will be no lint error:
```ts
interface Foo
{
  bar:
  {
    baz: string
  } }
```

**What did you expect to happen?**
Both the _class_ and _interface_ to be flagged by ESLint.

**What actually happened?**
The interface is _not_ flagged.

**Versions**

| package                     | version |
| --------------------------- | ------- |
| `@typescript-eslint/parser` | `1.13.0` |
| `TypeScript`                | `3.4.5` |
| `ESLint`                    | `6.0.1` |
| `node`                      | `8.11.4` |
| `npm`                       | `6.5.0` |
